src/cli/keepassxc-cli.cpp

```diff
@@ -130,6 +89,8 @@ int main(int argc, char** argv)
         parser.showHelp(EXIT_FAILURE);
     }
 
+    int exitCode = command->execute(argc, argv);
```

Comment:
@TheZ3ro the `argv` and `argc` have to be intact when instantiating the `QApplication`, otherwise the name of the app will be incorrect. I plan to only use `QCoreApplication` in the CLI in the future, so that we can instantiate it once in `keepassxc-cli`. This will require:
1- Using QProcess + native cli apps for clipping (already in progress)
2- Removing the gui prompt in favor of supporting a `--key-file` argument in all the CLI commands.

Until then,  we have to rely on duplicating the code in the commands...